.binder/postBuild

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 
 # Stop at errors
-set -e
+set -ex
 
 echo "### mybinder environment"
 env
 echo "### mybinder environment^^^"
+
+pip install .
```

Comment:
Nice :)

Reply:
hehe...

Reply:
I am currently waiting for binder to finish the installation, agreed it is very slow. I can't see from the environment why it should be slower than any other binder projects...

Reply:
now its working! :)

Reply:
I'm watching your build hahah, finished!

By the way, if we do `pip install .` then `requirements.txt` does not need to contain sisl's requirements right? Maybe it's better to remove them and keep there only the optional external ones, just to avoid inconsistencies and remove the need to update multiple files.

Reply:
Yeah, good idea, now changed.